lisa/messages.py

```diff
@@ -52,8 +52,12 @@ class TransportProtocol(str, Enum):
     Udp = "UDP"
 
 
+class CommunityTestMessage(MessageBase):
+    type: str = "CommunityTest"
+
+
 @dataclass
-class PerfMessage(MessageBase):
+class PerfMessage(CommunityTestMessage):
```

Comment:
Perf message is not related to CommunityTestMessage.

Reply:
So are we planning to move fio and ntttcp testcases to CommunityTestMessage as well? Or we do not change anything with fio and ntttcp testcases.

Reply:
That's different things. The ntttcp and fio are performance test cases, not functional test cases. The perf test cases supposed to pass every time, and we collect perf results. If it's failed, we should fix them. Because we care about the perf results.
For the community functional tests, one LISA test contains multiple one. It should report as one LISA test result and multiple community test results.